tests/test_single.py

```diff
@@ -238,6 +239,27 @@ def test_tx_status(http1_client):
     assert status == "Invalid"
 
 
+# pylint: disable=redefined-outer-name
+def test_openapi(http1_client):
+    """
+    Test that the openapi file is correct.
+    """
+    res = http1_client.raw().get("api")
+    assert res.status_code == HTTPStatus.OK
+    store_api = res.json()
+    print(store_api)
+
+    openapi_filename = "openapi.json"
+    logger.info("reading openapi.json")
+    with open(openapi_filename, "r", encoding="utf-8") as openapi_file:
+        file_openapi = json.load(openapi_file)
+    print(file_openapi)
+    with open(openapi_filename, "w", encoding="utf-8") as openapi_file:
+        logger.info("saving openapi.json file")
+        json.dump(store_api, openapi_file, indent=2, sort_keys=True)
+    assert store_api == file_openapi
```

Comment:
I think a nice error message here would be nice, as it's likely to happen if the API has been updated when a new feature is introduced.